### ISSUE TITLE
Roslaunch file

### ISSUE THREAD

**Issue by sam17** [KRSSG/grsim_comm#1]
Make a roslaunch file that runs this node with parameters of the publisher as input. Also, provide the option of not putting the parameters.

**Comment by korroktheslavemaster** [comment]
also, parameters of the grSim instance (IP, port). To be clear parameters of the publisher = topic name only?

**Comment by sam17** [comment]
No kickspeed and all as parameters for easier testing.

**Comment by korroktheslavemaster** [comment]
Kickspeed is part of the message itself

On Wed, Sep 2, 2015 at 8:25 PM, Dementor <notifications@github.com> wrote:

> No kickspeed and all as parameters for easier testing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KRSSG/grsim_comm/issues/1#issuecomment-137112814>.
>



-- 
Arpit Tarang Saxena
4th Year Undergraduate Student
Computer Science and Engineering
Indian Institute of Technology, Kharagpur
Ph: +91-7407650530, +91-9717585207

**Comment by sam17** [comment]
Yes so we can add parameters like that in launch file so that test_ssl node can use them and publish

**Comment by Vaishal-shah** [comment]
I will complete it by Sunday
